bleachbit/Action.py

```diff
@@ -165,8 +165,7 @@ def get_file(path):
 
         def get_walk_all(top):
             for expanded in glob.iglob(top):
-                for path in FileUtilities.children_in_directory(
-                        expanded, True):
+                for path in FileUtilities.children_in_directory(expanded, True):
```

Comment:
This looks like just a line break change? Usually I use `autopep` with the defaults. It's in the `makefile`.